vignettes/inboveg_queries.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "INBOVEG queries as functions"
+author: "Stijn Van Hoey"
+date: "March 23, 2018"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+library(knitr)
+library(tidyverse)
+library(DBI)
+opts_chunk$set(echo = TRUE)
+```
+
+# Introduction
+
+In this tutorial we show how to use helper functions to query data directly from the INBOVEG SQL Server database. These functions wrap **SQL queries** and so avoid having to write the queries yourself or copy-paste queries generated with the access-frontend for INBOVEG.
+
+We have provided functions to query information about surveys, metadata for relevés (= vegetation recording), the vegetation data and a query that helps to resolve taxonomic synonymy. 
+
+# The database connection
+
+In order to query data from INBOVEG, a database connection is required...
+
+You have to make sure you are allowed to set up this connection. For more information, see [link to more general database connection tutorial](...)...
+
+When the database name (the technical term is the [DSN](https://en.wikipedia.org/wiki/Data_source_name)) is known on your computer, you can make the connection as follows...
+
+```{r connection, eval = FALSE}
+con <- dbConnect(odbc::odbc(), dsn="Cydonia-prd") 
+```
```

Comment:
also connection to Futon-prd? now this is included in the 'synonym-solver' seperatly